package.json

```diff
@@ -55,6 +55,7 @@
     "react-paginating": "^1.4.0",
     "react-scripts": "5.0.0",
     "react-select": "^5.7.4",
+    "react-select-event": "^5.5.1",
```

Comment:
Where are you using this library?

Reply:
Ah, I found it…. the test file was collapsed (SubmissionConfig.test.js).

@nawatts also has https://github.com/DataBiosphere/terra-ui/pull/4032 up. I don't know if we have a preference for one approach over the other.

Reply:
Probably best to stay consistent and use the test-utils function. It's worth noting that the selectEvent tests had to be modified after React v18, making them more unwieldy. Two questions - what's the timeline for merging that PR, and does it still work properly with React 18?

@kpierre13 FYI

Reply:
I'm also unsure. One thing about `react-select-event` that I'm not keen on is that it looks like requires wrapping in `await act(async () => { ... })` when selecting a menu option triggers an async operation.

You could probably get away without either though with:
```ts
await user.click(selectMenu);
await user.click(screen.getByRole('option', { name: 'Option text' }))
```

The only downside there is that the option query is not scoped to the select menu's listbox, so if you happened to have multiple select menus with similar options open, it might select the wrong one.

Reply:
@nawatts do you want to resurrect 4032 and move it out of draft?

Reply:
Done. #4032 is ready for review.